.python-version

```diff
@@ -0,0 +1 @@
+3.7.0
```

Comment:
@speedlight there is already a [.tool-versions](https://github.com/stackbuilders/sb-debian-base/blob/master/.tool-versions) file in place, which flags the Python version used by this project. `asdf` has a feature called `legacy_version_file` which allows the tool to ready Python version from `.python-version`, if you consider worth it, we could turn it on for this project and get rid of `.tool-versions`, but maybe we could tackle that in a separate PR. What do you think about getting rid of this file for now?

https://asdf-vm.com/#/core-configuration?id=homeasdfrc

Reply:
Definetly! Sorry the for upload, this is not suposed to happend :(